content/knowledge_base/flow/cadence/accounts.md

```diff
@@ -0,0 +1,158 @@
+# Account
+
```

Comment:
Suggestion:
Add a single or two sentences about 'what' is an account in flow, and then move on to its specifics